CLAUDE.md

```diff
@@ -338,11 +362,12 @@ func (t *TestResource) Teardown(ctx context.Context) {
 - When testing a multi-step workflow, break it into individual state transitions
 
 Examples of state transition Context blocks:
-- `Context("'' -> Initializing")` - Initial reconciliation
-- `Context("Initializing -> Ready")` - Normal progression
-- `Context("Initializing -> Error")` - Error handling
-- `Context("Error -> ErrorBackoff")` - Recovery attempts
-- `Context("Ready -> Updating")` - Changes after ready state
+- `Context("'':'' -> Pending:Pending")` - Initial reconciliation
+- `Context("Pending:Pending -> Ready:Pending")` - Setup completion
+- `Context("Ready:Pending -> Ready:Running")` - Start processing
+- `Context("Ready:Running -> Ready:Succeeded")` - Successful completion
+- `Context("Ready:Pending -> Error:Failed")` - Error handling
+- `Context("Error:Failed -> Ready:Pending")` - Recovery attempts
```

Comment:
does it actually make sense to test status changes and phase changes in the same test? or should they be different?